src/components/GalleryLightBox/GalleryLightBox.tsx

```diff
@@ -10,8 +11,12 @@ import {
 
 import ImageElement from 'Image/ImageElement';
 import Icon from 'Icon';
+import Button from 'Button';
+
+import ViewportContext from 'ViewportContext/ViewportContext';
```

Comment:
YESSSSSS! It's being used outside of the header 😄

Reply:
it's a *very* handy piece of context